hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/writebehind/WriteBehindStore.java

```diff
@@ -50,10 +50,10 @@
     private final int partitionId;
 
     /**
-    * Number of issued flush operations.
-    * Flushes may be caused by an eviction. Instead of directly flushing entries
-    * upon eviction, the flushes are counted and immediately processed
-    * in {@link com.hazelcast.map.impl.mapstore.writebehind.StoreWorker}.
+     * Number of issued flush operations.
+     * Flushes may be caused by an eviction. Instead of directly flushing entries
+     * upon eviction, the flushes are counted and immediately processed
+     * in {@link com.hazelcast.map.impl.mapstore.writebehind.StoreWorker}.
      */
     private final AtomicInteger flushCounter;
```

Comment:
I would initialize this flushCounter + stagingArea in the class constructor. There is no point in doing in the 'regular' constructor. It only adds noise.